charts/argo-workflows/templates/controller/workflow-aggregate-roles.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   annotations:
     helm.sh/hook: pre-install
     helm.sh/hook-delete-policy: before-hook-creation
-  name: argo-workflows-aggregate-to-view
+  name: {{ template "argo-workflows.fullname" . }}-view
```

Comment:
These cluster roles are now specific to the release and as such don't have to be managed as hooks, so the hook annotations can be dropped.

Reply:
you are right, I dropped these now.